libs/ngrx-toolkit/src/lib/with-undo-redo.spec.ts

```diff
@@ -190,5 +191,33 @@ describe('withUndoRedo', () => {
         expect(store.canRedo()).toBe(true);
       });
     }));
+
+    it('clears undo redo stack', fakeAsync(() => {
```

Comment:
`fakeAsync` is going to be not available in zoneless and it is not required in this test. If you remove the `tick`, the test should still work.

Out of interest, why do you see the need for a `tick`?

Reply:
You are absolutely right. No need to have `fakeAsync` and `tick`. I used `tick` automatically based on my previous experiences with some not working tests in my projects. Sorry for this.